builder/java/deps.bzl

```diff
@@ -1,9 +1,12 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive", "http_jar")
 
 def deps():
+    RULES_JVM_EXTERNAL_TAG = "5.3"
+    RULES_JVM_EXTERNAL_SHA ="d31e369b854322ca5098ea12c69d7175ded971435e55c18dd9dd5f29cc5249ac"
+
```

Comment:
This would arguably be a good project-wide pattern for us to adopt.

`TAG` => `VERSION` would be a more descriptive name here.

Reply:
Can do, this is following the suggestion from the Bazel repository here: https://github.com/bazelbuild/rules_jvm_external/releases/tag/5.3